HUD/TakeControlOfFrontend.md

```diff
@@ -9,6 +9,20 @@ aliases: ["0xEC9264727EEC0F28"]
 void TAKE_CONTROL_OF_FRONTEND();
 ```
 
-Disables frontend (works in custom frontends, not sure about regular pause menu) navigation keys on keyboard. Not sure about controller. Does not disable mouse controls. No need to call this every tick.
+Disables frontend (works in both custom and regular pause menu) navigation keys on keyboard. Does not disable mouse controls. No need to call this every tick.
 
-To enable the keys again, use [`0x14621BB1DF14E2B2`](#_0x14621BB1DF14E2B2).
+To enable the keys again, use [`RELEASE_CONTROL_OF_FRONTEND`](#_0x14621BB1DF14E2B2).
+
+## Examples
+```lua
+-- Disable both navigation and mouse controls:
+
```

Comment:
Missing something?

Reply:
Nope, initially the comment was meant to tell what the code below was used for but I feel like it has no use now that I think about it